plugins/source/k8s/client/client.go

```diff
@@ -13,6 +13,7 @@ import (
 
 	// import all k8s auth options
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/rest"
```

Comment:
The changes in this file seems unrelated to this PR, can we do them in a separate one?
Otherwise the the k8s plugin will get a changelog entry with `feat: Add Resilience Hub resources`